AlphaWallet/Sell/ViewControllers/EnterSellTokensCardPriceQuantityViewController.swift

```diff
@@ -79,14 +79,14 @@ class EnterSellTokensCardPriceQuantityViewController: UIViewController, TokenVer
     private let tokenRowView: TokenRowView & UIView
     private let buttonsBar = HorizontalButtonsBar(configuration: .primary(buttons: 1))
     private var viewModel: EnterSellTokensCardPriceQuantityViewModel
-    private var totalEthCost: Ether {
+    private var totalEthCost: Double {
```

Comment:
I didn't check carefully, but isn't it better to pass around `Ether` than a `Double` for precision? Can we fix the formatting instead?

Also switching from `Ether` to `Double` would undo 4706f530a4a1431201e1dd29aea00f9a854f9583 which seems related to precision

```
commit 4706f530a4a1431201e1dd29aea00f9a854f9583
Author: Hwee-Boon Yar <hboon@motionobj.com>
Date:   2018-08-22 16:08:24 +0800

    Fix: ticket total ETH price should be precise
```